csfieldguide/static/interactives/number-memory/js/number-memory.js

```diff
@@ -71,13 +75,18 @@ function stall() {
 
 /**
  * Checks the number entered in the input box against the saved value.
- * Displays a message and reveals either the reset or reset-hard buttons depending on if the user is correct.
+ * Displays a message and reveals either the reset or reset-hard buttons depending on
+ * the mode and if the user is correct.
  */
 function submit() {
   showEndScreen();
   if ($('#number-input').val() == generatedNumber) {
     $('#middle-text').html('<span class="green">' + TXT_CORRECT + '</span>');
-    $('#reset-hard-button').removeClass('d-none');
+    if (isMemoryMode) {
+      $('#reset-button').removeClass('d-none');
+    } else {
+      $('#reset-hard-button').removeClass('d-none');
```

Comment:
Since there is now more than 2 modes I think we need to add a third button. 

Was strange to complete the 'hard mode' and then be prompted with 'hard mode' again - even though it's already in hard mode.

Reply:
I want it to be as much of a surprise as possible, so the user relaxes thinking it'll just repeat with a long number again. Also it's just a short joke interactive - people don't need to know there's more to it

Reply:
I think we should change the button to just say "Next level" instead of hard mode.

Also, can we please change the restart button to say 'Start again' and the colour be changed to the primary colour instead of red. Restart makes it sound like you are quitting and restarting half way through and red commonly implies something bad. This way is just comes across a bit more friendly.